app/src/main/cpp/skyline/common.h

```diff
@@ -22,6 +22,7 @@
 #include <memory>
 #include <compare>
 #include <variant>
+#include <random>
```

Comment:
You can remove the `#include <random>` from `common/uuid.cpp` which now has it redundantly included now that it's included here.

Reply:
Also make uuid use GetRandomBytes